### ISSUE TITLE
Genome Nexus changes the Reference Allele and Tumor_Seq_Allele2

### ISSUE THREAD

**Issue by thomasyu888** [genome-nexus/annotation-tools#30]
* Input
[input.txt](https://github.com/genome-nexus/annotation-tools/files/5859589/input.txt)

* annotation-tools intermediate files I must add the `.txt` at the end or github won't allow me to upload these.  My understanding it the `input.txt.temp.annotated.txt` is the output from Genome Nexus.  But because the annotation-tools allows us to include a directory with a list of mafs or vcfs, it annotates each one of those files separately.  `processed.txt` is all of these merged.
[input.txt.temp.annotated.txt](https://github.com/genome-nexus/annotation-tools/files/5859601/input.txt.temp.annotated.txt)
[input.txt.temp.txt](https://github.com/genome-nexus/annotation-tools/files/5859602/input.txt.temp.txt)

* Processed
[processed.txt](https://github.com/genome-nexus/annotation-tools/files/5859600/processed.txt)

**Comment by inodb** [comment]
Maybe not related but looks likeI end_position column is missing in the input? Not sure if that's the reason things go haywire tho

**Comment by thomasyu888** [comment]
Interesting. The input file doesn't have End_Position because we don't require centers to do it, but I made an edit to the original post to include End_Position

**Comment by thomasyu888** [comment]
pre process
```tsv
Tumor_Sample_Barcode	Chromosome	Start_Position	Reference_Allele	Tumor_Seq_Allele2
SAGE-1	10	89711833	A	AT
SAGE-1	10	89711833	A	ATT
```

post process

```tsv
Tumor_Sample_Barcode	Chromosome	Start_Position	End_Position	Reference_Allele	Tumor_Seq_Allele2
SAGE-1	10	89711833	89711834	-	A
SAGE-1	10	89711833	89711834	-	A
```

**Comment by sheridancbio** [comment]
This appears (on first glance) to be related to the issue of a shared common prefix in the alleles. A replacement of "A" with "AT" is (I think) semantically equivalent to an insertion of a "T" after the referenced "A". But the output above seems to be a bug of some sort. If a removal of a shared prefix had occurred, the offsets should have been adjusted. The first line has a shared prefix of CTTTTTTTTTTTT followed by either a T [reference] or a C [tumor]. Isn't this semantically equivalent to a SNP at the final position? So in that case , it should be a T->C SNP at position (170837513+13). I also notice that there are three input records here, but only 2 output records (post processing) I wonder what happened there.  @ao508 may know these rules off the top of her head.

**Comment by thomasyu888** [comment]
Thanks @sheridancbio and @inodb .  I made edits to my initial post. That is the complete list.

**Comment by ao508** [comment]
@thomasyu888 Would you mind slacking me the input MAFs with all of the fields? That would make it easier for me to troubleshoot the issue with the MAF standardization script we provided.

**Comment by thomasyu888** [comment]
@inodb @ao508 , This is ready to look at

**Comment by inodb** [comment]
Fixed in #39
